hackIDE/static/hackIDE/js/custom.js

```diff
@@ -269,7 +290,13 @@ $(document).ready(function(){
 						$(".output-io-info").hide();
 						$(".compile-status").children(".value").html("--");
 						$(".error-key").html("Compile error");
-						$(".error-message").html(response.compile_status);
+
```

Comment:
Hi!

Sure! How should I proceed? Two different branches with two different commits?

Should I create other branch solving the "empty error"?